src/main/java/review/ReviewInteractor.java

```diff
@@ -0,0 +1,89 @@
+package review;
+
+import entities.*;
+import recipe.RecipeRepoGateway;
+import recipe.RecipeRepoImpl;
+
+import java.io.IOException;
+
+public class ReviewInteractor {
+
+    private RecipeList recipeList;
+    private ReviewDatabase reviewDatabase;
+    private static final ReviewDatabaseReadWriter databaseReadWriter = new ReviewDatabaseReadWriter();
+
+
+    /**
+     * Construct a ReviewInteractor.
+     */
+    public ReviewInteractor() {
+        try {
+            RecipeRepoGateway rrg = new RecipeRepoImpl();
```

Comment:
RecipeRepoImpl is now RecipeReadWriter --> need to call RecipeReadWriter.getAllRecipes()